server/service/src/plugin/mod.rs

```diff
@@ -43,6 +44,15 @@ pub trait PluginServiceTrait: Sync + Send {
         uploaded_file.as_json_file(settings)
     }
 
+    fn reload_all_plugins(&self, ctx: &ServiceContext) -> Result<(), RepositoryError> {
+        let repo = BackendPluginRowRepository::new(&ctx.connection);
+        for row in repo.all()? {
```

Comment:
Just realised, that we don't have soft delete for plugins, Is there an issue for this, I think we definitely want to be able to remove them...

Reply:
Agree, was on my list but not captured, thanks: https://github.com/msupply-foundation/open-msupply/issues/6448